setup.py

```diff
@@ -111,7 +111,7 @@
             'gcode_tempcal=gcodeutils.gcode_tempcal:main',
             'gcode_mod=gcodeutils.gcode_mod:main',
             'gcode_stretch=gcodeutils.gcode_stretch:main',
-            'gcode_arc_optimize=gcodeutils.gcode_optimize_arcs:main',
+            'gcode_optimize_arc=gcodeutils.gcode_optimize_arcs:main',
```

Comment:
maybe it would also be better to also rename the py script to gcode_optimize_arc.py (without the last "s")